app/api/routes.py

```
@@ -400,5 +404,73 @@ def delete_record_route(
+@router.post("/api/v1/consult/chat",
+     tags=["에이전트 상담"],
+     summary="에이전트 대화",
+     description="세션 ID를 사용하여 에이전트와 대화를 나눕니다. Server-Sent Events(SSE) 형식으로 스트리밍 응답을 반환합니다.",
+     responses = {
+         200: {
+            "description": "에이전트 응답 스트림",
+            "content": {"text/event-stream": {"example": "응답 텍스트가 실시간으로 스트리밍됩니다."}}
+         }
+     }
+)
+def send_chat_message(request: ChatRequest, db: Session = Depends(get_db), current_user: User = Depends(AuthTokenDep)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)